docs/linking/README.md

```diff
@@ -262,7 +317,7 @@ device, the PISP, and the DFSP.
 To make this work, we simply need to provide a way for a member on the network
 to remove the Consent resourse and notify the other parties about the removal.
 
-![Credential registration: Register](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/mojaloop/pisp/master/docs/linking/unlinking.puml)
+![Unlinking](../out/linking/unlinking.svg)
```

Comment:
what about unregistering THIRD_PARTY_LINK entries?

Reply:
Good point, I'll follow that up in the next pr